src/tools/illink/test/Mono.Linker.Tests.Cases/Attributes/OnlyKeepUsed/NullableOnConstraintsRemoved.cs

```diff
@@ -1,5 +1,8 @@
 #nullable enable
 
+using System.Reflection;
+using System.Runtime.CompilerServices;
+
 using Mono.Linker.Tests.Cases.Expectations.Assertions;
```

Comment:
nit: remove blank line

```suggestion
using System.Reflection;
using System.Runtime.CompilerServices;
using Mono.Linker.Tests.Cases.Expectations.Assertions;
```